src/gleam_stdlib.mjs

```diff
@@ -633,6 +635,9 @@ export function decode_map(data) {
   if (data instanceof PMap) {
     return new Ok(PMap.fromMap(data));
   }
+  if (data == null) {
```

Comment:
Could we also return this error if the typeof data is not object? It's still possible to crash this on older runtimes otherwise.

Reply:
done